examples/04_manipulating_images/plot_multiply_image.py

```diff
@@ -0,0 +1,21 @@
+"""
+Multiply image
+==============
+
+Here we multiply the voxel values of an image by -1.
+"""
+
+from nilearn import datasets, plotting, image
+
+data = datasets.fetch_atlas_aal()
+
+# Print basic information on the AAL regions.
+print('AAL regions nifti image (3D) is located at: %s' % data.regions)
```

Comment:
Minor comment: Could you just use heading notations like other examples ?
starting with ############################# gives good consistency with other examples we have.